examples/simultaneous_translation/data/ast_dataset.py

```diff
@@ -64,47 +56,33 @@ def __getitem__(self, index):
         path = self.aud_paths[index]
         if not os.path.exists(path):
             raise FileNotFoundError("Audio file not found: {}".format(path))
```

Comment:
Note that is your features and wav files are not side by side, this will raise an error even though the feature file exists.

Reply:
Any comment on this one?